OrbitGl/DataManager.h

```diff
@@ -50,6 +54,7 @@ class DataManager final {
   absl::flat_hash_set<uint64_t> visible_functions_;
   int32_t selected_thread_id_ = -1;
   const TextBox* selected_text_box_ = nullptr;
+  std::shared_ptr<Process> selected_process_ = std::make_shared<Process>();
```

Comment:
nit: maybe also a TODO comment that we want to get rid of "OrbitProcess" soon. That is the plan right? It feels somewhat wrong to use OrbitProcess in DataManager, but I understand it is necessary.

Reply:
I added a todo.